backend/src/test/java/com/benine/backend/PresetControllerTest.java

```diff
@@ -43,6 +43,20 @@ public void testAddPreset() throws Exception {
   }
   
   @Test
+  public void testUpdatePreset() throws Exception {
```

Comment:
I think most of the code in these tests can be put in an `@Before` statement. It cleans up a lot of duplicate code in the tests.